examples/multiplatform/MacOSApp/Example/BUILD

```diff
@@ -50,7 +50,7 @@ swift_library(
     module_name = "iOSApp",
     tags = ["manual"],
     deps = [
-        "//examples/macos_app/third_party:ExampleFramework",
+        "//examples/multiplatform/MacOSApp/third_party:ExampleFramework",
```

Comment:
Can we capitalize it as `macOSApp`? Since you probably have a case insensitive filesystem, you might have to do some odd things in git to have it pickup the rename: https://stackoverflow.com/a/40307511 or otherwise